package-lock.json

```diff
@@ -18,6 +18,7 @@
         "graphql-tag": "~2.12.6",
         "react": "~17.0.2",
         "react-dom": "~17.0.2",
```

Comment:
Please remove package lock json changes from pull request

Reply:
@Katsu08 Do you mean remove those

 "graphql-tag": "~2.12.6",
 "react": "~17.0.2",
"react-dom": "~17.0.2",

 three packages?

Reply:
I meant that package lock json was changed in this pull request, but that file doesn't should change hehe. You can copy the original content from dev branch and paste in your branch and push again:

https://github.com/Azordev/maps-project/blob/dev/package-lock.json